src/XRP/model/xrp-account-delete.ts

```diff
@@ -35,7 +39,10 @@ export default class XrpAccountDelete {
       xrplNetwork == XrplNetwork.Test,
```

Comment:
Should this include ` || xrplNetwork == XrplNetwork.Dev`?

Reply:
i believe so. @amiecorso did we ever add a function that combines the XrplNetwork.Test and XrplnetWork.Dev checks?